content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -152,8 +180,10 @@ The Istio control plane must be installed with virtual machine integration enabl
     [Determining the ingress host and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) to set the environment variable `${INGRESS_HOST}`.
 
     {{< text bash >}}
-    $ touch "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/hosts-addendum
-    $ echo "${INGRESS_HOST} istiod.istio-system.svc" > "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/hosts-addendum
+    $ touch "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/hosts
+    $ echo "${INGRESS_HOST} istiod.istio-system.svc" >> "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/hosts
+    $ echo "1.1.1.1 pod.${SERVICE_NAMESPACE}.svc.cluster.local" >> "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/hosts
```

Comment:
why 1.1.1.1? I know this is a google root name service, however, the purpose of the 1.1.1.1 above is to generate an error - as to obtain the service CIDR.

Reply:
Strongly recommend decoupling DNS from setup. We can have a section on various DNS options like https://learn.hashicorp.com/tutorials/consul/dns-forwarding?in=consul/security-networking (consul docs are great). For now, to make this PR viable, I strongly suggest leaving DNS as an exercise left to the reader

Reply:
Do we want to put a service IP for a k8s workload, just to enable the demo at the end of the PR? We can add a link to the DNS section, which can be added later (a separate PR).

Reply:
will add a seperate PR for this. Currently set using a fix one

Reply:
A demo should not be in this PR. Demos belong in the examples path of the repository.